pouchdb-core/index.d.ts

```diff
@@ -426,6 +431,10 @@ declare namespace PouchDB {
         on(event: 'created', listener: (dbName: string) => any): this;
         on(event: 'destroyed', listener: (dbName: string) => any): this;
 
+        debug: debug.IDebug;
+
+        adapter(id: string, adapter: any, addToPreferredAdapters?: boolean): void;
```

Comment:
It's used for adding new plugable adapters. It's not really documented on the Pouch API page, so was trying to get some feedback from the developers to whether this is an official API or not.